lib/shoulda/matchers/active_record/define_enum_for_matcher.rb

```diff
@@ -84,6 +84,8 @@ def description
             desc << " with #{options[:expected_enum_values]}"
           end
 
+          desc << " and store the value in a column with an integer type"
```

Comment:
It's a _little_ verbose, but I didn't want to have two different messages on the same matcher.